### DISCUSSION TITLE
Bindings issue with MongoDB

### DISCUSSION THREAD

**Discussion by cyl1234567** [Budibase/budibase#6771] [Help]
Hi, Every one

I can run a MongoDB update directly. But when I use bindings, the system always toast a query error.
("ID" is a field in numeric format)

Is this a bug or am I missing a specific escape character?

app version: 1.1.7

screen shot:
- query directly
  - <img width="527" alt="image" src="https://user-images.githubusercontent.com/109498463/179445455-7bf5ee52-2fa1-4e37-9677-f4e3234753f6.png">
- use bindings
  - <img width="522" alt="image" src="https://user-images.githubusercontent.com/109498463/179445831-bae8d99a-e4a6-4d8d-b526-1d6a0920cf95.png">
  - <img width="321" alt="image" src="https://user-images.githubusercontent.com/109498463/179446305-e37ab1c4-975d-4144-b83c-5c166a8ffb1b.png">

**Comment by melohagan** [comment]
Hi @cyl1234567 

Thanks for highlighting. I was able to reproduce and I've raised a bug here: https://github.com/Budibase/budibase/issues/6778
